server.go

```diff
@@ -883,21 +897,7 @@ func (s *Server) handleStream(t transport.ServerTransport, stream *transport.Str
 	method := sm[pos+1:]
 	srv, ok := s.m[service]
 	if !ok {
-		if trInfo != nil {
-			trInfo.tr.LazyLog(&fmtStringer{"Unknown service %v", []interface{}{service}}, true)
```

Comment:
I was saying the content of the string for `LazyLog` is wrong... It will only say "unknown method", even in the case of "unknown service".

There's no requirement on the description. But I would prefer to keep it unchanged, in case someone is doing string matching on that.